Packs/MicrosoftGraphMail/Integrations/MicrosoftGraphMail/pipfile

```diff
@@ -19,7 +19,7 @@ networkx = "==2.5"
 beautifulsoup4 = "==4.9.3"
 freezegun = "==0.3.12"
 vulture = "==1.5"
-parinx = {ref = "6493798ceba8089345d970f71be4a896eb6b081d", git = "https://github.com/coyo8/parinx.git"}
+parinx = {ref = "6493798ceba8089345d970f71be4a896eb6b081d", git = "https://github.com/coyo8/parinx.git", egg = "parinx"}
```

Comment:
What is this change?

Reply:
Something I needed for the testing, i'll revert it